main/station-config.js

```diff
@@ -73,35 +72,13 @@ function setTrayOperationExplained () {
   configStore.set(ConfigKeys.TrayOperationExplained, TrayOperationExplained)
 }
 
-/**
- * @returns {string | undefined}
- */
-function getFilAddress () {
-  return FilAddress || 'f0111' // needs refactor - same as getStationWalletAddress
```

Comment:
Is it worth our effort to implement a migration that will read the FIL address provided by the user and set it as the destination address in the wallet?

Reply:
I actually don't think so. It could be confusing for the user even. If we don't add it, everyone will go through the same flow to set up their wallet.